day_1/project_of_the_day.py

```diff
@@ -0,0 +1,18 @@
+# https://replit.com/@appbrewery/band-name-generator-end
+
+GREETING = 'Welcome to the Band Name Generator.'
+ASK_CITY = 'What`s the name of the city you grew up in?'
+ASK_PET_NAME = 'What`s your pet`s name?'
+NAME_COMPONENTS = (ASK_CITY, ASK_PET_NAME)
+TELL_BAND_NAME = 'Your band name could be'
+SPACE = ' '
+
+print(GREETING)
+
+band_name = ''
+for component in NAME_COMPONENTS:
+    band_name += input(component + '\n')
+    band_name += SPACE
+band_name.rstrip(SPACE)
+
+print(TELL_BAND_NAME, band_name)
```

Comment:
It depends on the task, but generally I think it's not a good idea to print output this way, because it prints not a formatted string, but a tuple instead.
I think it'd be to use `TELL_BAND_NAME` as a template (e.g. `TELL_BAND_NAME = 'foo {}'`) and then print a string formatted into a template: `TELL_BAND_NAME.format(band_name)`.

Reply:
OK, I`ll fix it!